assets/src/components/dashboard/Overview.js

```
@@ -127,10 +127,10 @@ const Overview = () => {
                       </p>
                       <span className="feature-requests-submit-button">
                         <a
-                          href={baseUrlStoregrowth + links?.featureRequest}
+                          href={`https://storegrowth.io/contact-us/`}
                           target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)